documentation/vts/index.md

```diff
@@ -0,0 +1,5 @@
+# Developer Documentation
+
+This documentation was generated from the XML comments within the source files. We also have a downloadable CHM file that replicates the MSDN style documentation from Microsoft.
+
+That file can be downloaded [here](https://github.com/VirtualPhotonics/VTS/releases/download/Version_8.0/VtsDocumentation.chm)
```

Comment:
This documentation will become out of sync. Now that it's on the web, can we drop the file creation? IMO anything we can do to have tools automate our release process can help make releases easier/safer/cheaper to do.